requirements/edx/paver.txt

```diff
@@ -5,3 +5,4 @@ lazy==1.1
 path.py==7.2
 watchdog==0.7.1
 python-memcached
+libsass==0.10.0
```

Comment:
It generates `/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/_sass.so` in devstack.